graphics/fontforge/Portfile

```diff
@@ -165,6 +177,6 @@ variant gui description {Enable GUI support} {
 }
 
 default_variants    +gui
-if {![variant_isset python38] && ![variant_isset python39] && ![variant_isset python310] && ![variant_isset python311]} {
+if {![variant_isset python38] && ![variant_isset python39] && ![variant_isset python310] && ![variant_isset python311] && ![variant_isset python312]} {
     default_variants-append +python311
```

Comment:
Perhaps the default variant should now be 3.12?

Reply:
Okay. Consider Python 3.12 is the latest stable version released.